pkg/launcher/uninstall/uninstall_linux.go

```diff
@@ -0,0 +1,23 @@
+package uninstall
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/kolide/launcher/ee/allowedcmd"
+)
+
+func disableAutoStart(ctx context.Context) error {
+	// the --now flag will disable and stop the service
+	cmd, err := allowedcmd.Systemctl(ctx, "disable", "--now", "launcher.kolide-k2.service")
```

Comment:
This will only work with systemd. Which is _most_ but not all of our systems. OTOH, I think this this disable function isn't going to last for more than a release so 🤷